src/test/java/org/folio/edge/sip2/api/MainVerticleTests.java

```diff
@@ -79,8 +81,38 @@ public void canStartMainVericleInjectingSip2RequestHandlers(Vertx vertex,
   @Test
   public void cannotCheckoutWithInvalidCommandCode(Vertx vertex,
       VertxTestContext testContext) throws Throwable {
-    callService("blablabalb", testContext, vertex, result -> {
+      callService("blablabalb", testContext, vertex, result -> {
       assertTrue(result.contains("Problems handling the request"));
     });
   }
+
+  @Test
+  public void canMakeValidSCStatusRequest(Vertx vertex,
+                                          VertxTestContext testContext) throws Throwable {
+
+    callService("9900401.00AY1AZFCA5",
+      testContext, vertex, result -> {
+        String expectedPreLocalTime = "98YYNYNN53" + getFormattedDateString();
+        String expectedPostLocalTime = "1.23|AOfs00000010test|AMChalmers|BXYNNNYNYNNNNNNNYN|ANTL01|AFscreenMessages|AGline\n\n";
```

Comment:
Is this an actual response or is something else supposed to happen to it? `AGline\n\n` can't be right? It'd have to be `AGline|\r` to be correct, right?

Reply:
I can only get the template to produce "\n" by virtue of  default carriage return. (\r is used in Mac OS environment. \n is Linux, \r\n is Windows.  Hope we don't have to manually create the template on a Mac :))